.gitignore

```diff
@@ -1,2 +1,3 @@
 *.gem
+.ruby-version
```

Comment:
I had to downgrade ruby to be compatible with the commited Gemfile.lock so this should be useful so people can set their ruby without risking commiting the file.